test/exe/Main.hs

```diff
@@ -1061,31 +1061,33 @@ completionTests
         let source = T.unlines ["module A where", "f = hea"]
         docId <- openDoc' "A.hs" "haskell" source
         compls <- getCompletions docId (Position 1 7)
-        liftIO $ compls @?= [complItem "head" ["GHC.List", "base", "v", "head"] (Just CiFunction)]
+        liftIO $ map dropDocs compls @?= [complItem "head" ["GHC.List", "base", "v", "head"] (Just CiFunction) (Just "[a] -> a")]
     , testSessionWait "type" $ do
         let source = T.unlines ["{-# OPTIONS_GHC -Wall #-}", "module A () where", "f :: ()", "f = ()"]
         docId <- openDoc' "A.hs" "haskell" source
         expectDiagnostics [ ("A.hs", [(DsWarning, (3,0), "not used")]) ]
         changeDoc docId [TextDocumentContentChangeEvent Nothing Nothing $ T.unlines ["{-# OPTIONS_GHC -Wall #-}", "module A () where", "f :: Bo", "f = True"]]
         compls <- getCompletions docId (Position 2 7)
-        liftIO $ compls @?=
-            [ complItem "Bounded" ["GHC.Enum", "base", "t", "Bounded"] (Just CiClass)
-            , complItem "Bool" ["GHC.Types", "ghc-prim", "t", "Bool"] (Just CiClass)
+        liftIO $ map dropDocs compls @?=
+            [ complItem "Bounded" ["GHC.Enum", "base", "t", "Bounded"] (Just CiClass) Nothing
+            , complItem "Bool" ["GHC.Types", "ghc-prim", "t", "Bool"] (Just CiClass) Nothing
             ]
     , testSessionWait "qualified" $ do
         let source = T.unlines ["{-# OPTIONS_GHC -Wunused-binds #-}", "module A () where", "f = ()"]
         docId <- openDoc' "A.hs" "haskell" source
         expectDiagnostics [ ("A.hs", [(DsWarning, (2, 0), "not used")]) ]
         changeDoc docId [TextDocumentContentChangeEvent Nothing Nothing $ T.unlines ["{-# OPTIONS_GHC -Wunused-binds #-}", "module A () where", "f = Prelude.hea"]]
         compls <- getCompletions docId (Position 2 15)
-        liftIO $ compls @?= [complItem "head" ["GHC.List", "base", "v", "head"] (Just CiFunction)]
+        liftIO $ map dropDocs compls @?= [complItem "head" ["GHC.List", "base", "v", "head"] (Just CiFunction) (Just "[a] -> a")]
     ]
   where
-    complItem label xdata kind = CompletionItem
+    dropDocs :: CompletionItem -> CompletionItem
+    dropDocs ci = ci { _documentation = Nothing }
+    complItem label xdata kind ty = CompletionItem
       { _label = label
       , _kind = kind
-      , _detail = Just "Prelude"
-      , _documentation = Just (CompletionDocMarkup (MarkupContent {_kind = MkMarkdown, _value = ""}))
+      , _detail = (":: " <>) <$> ty
```

Comment:
This should also check `optNewColonConvention`.

Reply:
I am not sure how to do this, since I don't know where the tests set this flag.

Reply:
Oh I missed that this is a test. Fine to leave it in that case since the test will always use double colons.